src/highdicom/pr/content.py

```diff
@@ -681,17 +702,26 @@ def _check_coords(
             )
 
 
-class SoftcopyVOILUT(VOILUT):
+class SoftcopyVOILUTTransformation(VOILUTTransformation):
```

Comment:
I thought that you wanted to change this so that it no longer subclasses VOILUTTransformation but rather takes one as a parameter?

I think it's ok as is but up to you

Reply:
Yes, I initially thought it may be cleaning to call it `SoftcopyVOILUT` to make clear that it's an item of the Softcopy VOI LUT Sequence. However, it's more closely related to `VOILUTTranslation` than to `VOILUT` and therefore the name would be confusing.

The [Softcopy VOI LUT Module](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.11.8.html) contains the [VOI LUT Macro](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.11.2.html#table_C.11-2b) and additional attributes. Conceptually, the `SoftcopyVOILUTTransformation` is a special subtype of `VOILUTTransformation` that potentially restricts the transformation to a subset of images.